students/mariusz_michalczyk/lesson_07_strings/date_printer.py

```diff
@@ -0,0 +1,7 @@
+from datetime import datetime
+
+
+if __name__ == '__main__':
+    now = datetime.now()
+    print("Today is {0} year {1} month and {2} day. Time is: {3}".
```

Comment:
You can use strftime for getting month name also, which would fit "human readable" description a bit more.